src/components/AddPaymentMethodMenu.js

```diff
@@ -1,15 +1,18 @@
-import React from 'react';
 import PropTypes from 'prop-types';
+import React from 'react';
 import {withOnyx} from 'react-native-onyx';
+
+import Permissions from '@libs/Permissions';
```

Comment:
NAB Seems weird that `Permissions` comes before `compose`

Reply:
https://github.com/trivago/prettier-plugin-sort-imports#importordercaseinsensitive Reason for this issue.

Reply:
Can we please set [`importordercaseinsensitive`](https://github.com/trivago/prettier-plugin-sort-imports#importordercaseinsensitive) to `true`?